mkosi

```diff
@@ -1626,6 +1756,30 @@ def find_postinst_script(args):
     if os.path.exists("mkosi.postinst"):
         args.postinst_script = "mkosi.postinst"
 
+def find_passphrase(args):
+
+    if args.encrypt is None:
+        args.passphrase = None
+        return
+
+    try:
+        passphrase_mode = os.stat('mkosi.passphrase').st_mode & (stat.S_IRWXU | stat.S_IRWXG | stat.S_IRWXO)
+        if (passphrase_mode & stat.S_IRWXU > 0o600) or (passphrase_mode & (stat.S_IRWXG | stat.S_IRWXO) > 0):
+            print_step("Permissions of 'mkosi.passphrase' of '{}' are too open. Please make sure that the file is only accessible by its owner. Aborting.".format(oct(passphrase_mode)))
```

Comment:
I don’t really love this phrasing – it sounds like the solution is to `chmod` the file, but it would be better to generate a new passphrase (giving it the right permissions from the start), since an adversary might already have read the current passphrase file. Is there a good way to express that without making the error message too long?

Also, all the other `sys.exit`s are preceded by `sys.stderr.write` instead of `print_step` – error messages aren’t steps. (There’s enough of them that it might be a good idea to make a `die(message)` function, but that’s for another PR.)